Testing/Temporary/CTestCostData.txt

```diff
@@ -0,0 +1 @@
+---
```

Comment:
what is it file for?

Reply:
I don't know - this code was auto generated..

Reply:
Well, then it should not be here.

Reply:
Remove this file?